app/src/features/admin/users/AccessRequestList.tsx

```diff
@@ -83,6 +86,28 @@ const AccessRequestList: React.FC<IAccessRequestListProps> = (props) => {
 
     setActiveReviewDialog({ open: false, request: null });
 
+    try {
+      await biohubApi.admin.sendGCNotification(
+        {
+          emailAddress: updatedRequest.data.email,
+          userId: updatedRequest.id
+        } as IgcNotifyRecipient,
+        {
+          subject: 'SIMS: Your request for access has been approved.',
+          header: 'Your request for access to the Species Inventory Management System has been approved.',
+          body1: 'This is an automated message from the BioHub Species Inventory Management System',
+          body2: '',
+          footer: ''
+        } as IgcNotifyGenericMessage
```

Comment:
Is this cast `as IgcNotifyGenericMessage` necessary? 

This should be able to match whatever interface we've defined without having to force it.
Same with the cast above.

Reply:
Or possibly the interfaces need updating, if they aren't sufficiently handling all use cases?